### ISSUE TITLE
use tensorboard visualizations

### ISSUE THREAD

**Issue by Tilana** [Tilana/cnn#3]
visualize learning rate, accuracy, and tensor graph (code structure) with tensorboard

**Comment by Tilana** [comment]
Tensorflow Guide: https://www.tensorflow.org/versions/r0.12/how_tos/graph_viz/

**Comment by Tilana** [comment]
tensorboard --logdir=path/to/logfiles

**Comment by Tilana** [comment]
This issue was moved to Tilana/Classification#68
